LondonBritishLibrary/orient/BLorient764.xml

```diff
@@ -25,26 +25,452 @@ schematypens="http://relaxng.org/ns/structure/1.0"
+<textLang mainLang="gez"></textLang>
+</msItem>
+
+<msItem xml:id="ms_i2.29">
+<locus from="182r"/>
+<title type="complete" ref="LIT4838CollectioM#Chapter29">John Chrysostom, <foreign xml:lang="gez">ዘዮሐንስ፡ ጳጳስ፡ ዘቈስጠንጢኖጶሊስ፡ በእንተ፡ አርምሞ፡ ዘተናገረ፡</foreign></title>
+<textLang mainLang="gez"></textLang>
+</msItem>
+
+<msItem xml:id="ms_i2.30">
+<locus from="182v"/>
+<title type="complete" xml:lang="en" ref="LIT4838CollectioM#Chapter30">Another extract from Chrysostom</title>
+<textLang mainLang="gez"></textLang>
+</msItem>
+
+<msItem xml:id="ms_i2.31">
+<locus from="183v"/>
+<title type="complete" xml:lang="en" ref="LIT4838CollectioM#Chapter31">Peter, the father of Ephraim</title>
+<textLang mainLang="gez"></textLang>
+</msItem>
+
+<msItem xml:id="ms_i2.32">
+<locus from="184v"/>
+<title type="complete" xml:lang="en" ref="LIT4838CollectioM#Chapter32">Anonymous</title>
+<textLang mainLang="gez"></textLang>
+</msItem>
+
+<msItem xml:id="ms_i2.33">
+<locus from="185v"/>
+<title type="complete" xml:lang="en" ref="LIT4838CollectioM#Chapter33">Anonymous</title>
+<textLang mainLang="gez"></textLang>
+</msItem>
+
+<msItem xml:id="ms_i2.34">
+<locus from="185v"/>
+<title type="complete" xml:lang="en" ref="LIT4838CollectioM#Chapter34">Anonymous</title>
+<textLang mainLang="gez"></textLang>
+</msItem>
+
+<msItem xml:id="ms_i2.35">
+<locus from="187r"/>
+<title type="complete" xml:lang="en" ref="LIT4838CollectioM#Chapter35">ʾAbbā Moses</title>
+<textLang mainLang="gez"></textLang>
+</msItem>
+
+<msItem xml:id="ms_i2.36">
+<locus from="189r"/>
+<title type="complete" ref="LIT4838CollectioM#Chapter36">ትእዛዘ፡ ወገድል፡ ዘአእሩግ፡</title>
+<textLang mainLang="gez"></textLang>
+</msItem>
+
+<msItem xml:id="ms_i2.37">
+<locus from="189v"/>
+<title type="complete" ref="LIT4838CollectioM#Chapter37">ይቤሉ፡ አእሩግ፡</title>
+<textLang mainLang="gez"></textLang>
+</msItem>
+
+<msItem xml:id="ms_i2.38">
+<locus from="191r"/>
+<title type="complete" ref="LIT4838CollectioM#Chapter38">Paul</title>
+<textLang mainLang="gez"></textLang>
+</msItem>
+
+<msItem xml:id="ms_i2.39">
+<locus from="192v"/>
+<title type="complete" xml:lang="gez" ref="LIT4838CollectioM#Chapter39">ዘተስእሉ፡ ወነገርዎም፡ ለሊቃውንት፡ በእንተ፡ ሕሊና፡ ዝሙት፡</title>
+<textLang mainLang="gez"></textLang>
+</msItem>
+
+<msItem xml:id="ms_i2.40">
+<locus from="195v"/>
+<title type="complete" xml:lang="gez" ref="LIT4838CollectioM#Chapter40">መጽሐፍ፡ ዘ፩፡ እምአበው፡ ቀዳምት፡ መሃይምናን፡</title>
+<textLang mainLang="gez"></textLang>
+</msItem>
+
+<msItem xml:id="ms_i2.41">
+<locus from="198r"/>
+<title type="complete" xml:lang="gez" ref="LIT4838CollectioM#Chapter41">ነገር፡ ዘተናገረ፡ ፩፡ እምሥዩማነ፡ ቤተ፡ ክርስቲያን፡ በመዋዕለ፡ ዐባር፡ ወራኃብ፡</title>
+<textLang mainLang="gez"></textLang>
+</msItem>
+
+<msItem xml:id="ms_i2.42">
+<locus from="199r"/>
+<title type="complete" xml:lang="gez" ref="LIT4838CollectioM#Chapter42">ዝንቱ፡ ነገር፡ ዘተናገረ፡ ፩፡ ካህን፡ በምድረ፡ ካም፡ ዘበምንኵስናሁ፡ ተሠይመ፡ ውስተ፡ ክህነት፡ ነቢሮ፡ ውስተ፡ ምኔታት፡ ፲ዓመተ፡ በደሮቲዮስ፡ ሊቀ፡ ጳጳሳት፡</title>
```

Comment:
maybe move this into the incipit?

Reply:
I don't really see how to divide into title and incipit. Where would you put the pause?

Reply:
I would probably move it to the incipit completely and then take the conventional English title, if available for the title